openai/openai.go

```diff
@@ -367,6 +403,18 @@ func fromChatRequest(r ChatCompletionRequest) (*api.ChatRequest, error) {
 			}
 			messages = append(messages, message)
 		default:
+			if msg.Role == "assistant" && msg.ToolCalls != nil {
```

Comment:
Do we need to check for `assistant` here?

Reply:
I believe we can avoid this `if` altogether since we're ranging over `msg.ToolCalls`